skills/strategy-marketplace/SKILL.md

```
@@ -0,0 +1,219 @@
+- Offer inference/strategy services to other agents
+
+## Installation
+
+```bash
+# This skill integrates with starknet-agentic
+# Clone into your skills directory
+```
+
+## Capabilities
+
+### 1. Register Agent
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)